### ISSUE TITLE
Convert to standard test suite

### ISSUE THREAD

**Issue by nimbinatus** [deconst/preparer-sphinx#75]
Looks like the tests here are custom tests (@smashwilson, am I missing something/not recognizing the testing suite here?).

**Comment by smashwilson** [comment]
Nope, you're not missing anything. I think I had trouble mocking enough of Sphinx's innards, so I wrote some integration-style tests that basically diff the JSON output? It's been a while :smile:
